src/main/groovy/com/twcable/grabbit/client/batch/steps/jcrnodes/JcrNodesWriter.groovy

```diff
@@ -71,7 +70,7 @@ class JcrNodesWriter implements ItemWriter<ProtoNode>, ItemWriteListener {
     void write(List<? extends ProtoNode> nodeProtos) throws Exception {
         Session session = theSession()
         for (ProtoNode nodeProto : nodeProtos) {
-            writeToJcr(nodeProto, session)
+            new ProtoNodeDecorator(nodeProto).writeJcrNodesRecursively(session)
```

Comment:
agreed.. I wanted to differentiate this method from other write to jcr and this found this name relevant to what the method is doing... will change it to something more appropriate